docs/index.rst

```diff
@@ -75,20 +99,85 @@ An example of how to read positions from a TNG file is shown below.
          # get the data from the requested block by supplying NumPy array which
          # is updated in-place or returned 
          
-         positons = ts.get_positions(positions)
+         # update in place
+         ts.get_positions(positions)
+         # positions = ts.get_positions(positions) is equivalent
+
+         # or return by value
+         box_vec = ts.get_box(box_vec)
+         # ts.get_box(box_vec) is equivalent
```

Comment:
Stay consistent... I was confused "What, this one returns a value and the other does in place?" until I read the comment. Just make both code examples "in place" and then show in the comments the return value form.